.github/workflows/compile-arduino.yml

```diff
@@ -33,6 +33,7 @@ jobs:
             - name: FreeRTOS
             - name: SimpleRotary
             - name: LiquidCrystal I2C
+            - name: printf
```

Comment:
```suggestion
            - source-url: https://github.com/mpaland/printf.git
              version: latest
```

Reply:
This library is needed only for `float` ssupport, suppose we can implement flat specialization of control manually, get existing implementation.